yarn-project/bb-prover/src/bb/execute.ts

```diff
@@ -412,6 +416,7 @@ async function verifyProofInternal(
   verificationKeyPath: string,
   command: 'verify' | 'avm_verify',
   log: LogFn,
+  calldataPath?: string,
```

Comment:
to keep this function generic it's probably better to have instead "extraArgs?: [string]"